config/application.rb

```diff
@@ -17,7 +23,8 @@ module ServiceManualPublisher
   GDS_ORGANISATION_CONTENT_ID = "af07d5a5-df63-4ddc-9383-6a666845ebe9".freeze
 
   class Application < Rails::Application
-    config.load_defaults 6.0
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 6.1
```

Comment:
We usually make this change in a separate commit, after testing that each of the new defaults works. I think including it in the `app:update` commit has a bit too much going on.

Reply:
You should also delete the [6.1 defaults file](https://github.com/alphagov/service-manual-publisher/blob/d5695aea03a8a0b6c6bc15523f8d85ddf80d1f6c/config/initializers/new_framework_defaults_6_1.rb) as part of this commit.

(I think the correct process is to uncomment each config option of that file and run the tests, ideally committing as you go. The change of `load_defaults 6.1` should be the final step).